packages/fxa-payments-server/src/index.tsx

```diff
@@ -14,6 +14,7 @@ import { actions } from './store/actions';
 
 async function init() {
   readConfigFromMeta(headQuerySelector);
+
```

Comment:
Added a whitespace change to payments-server to cause a Storybook build. I can always back this out